### DISCUSSION TITLE
Overlay frames from a gif into a still image

### DISCUSSION THREAD

**Discussion by hellocatfood** [ImageMagick/ImageMagick#2422] [Help] [answered]
I have this animated gif

![animation](https://user-images.githubusercontent.com/151947/90334383-317c4300-dfc5-11ea-9120-6d3f0143b943.gif)

And I want to combine each frame of it into a still image, like below (made in GIMP)

![animation](https://user-images.githubusercontent.com/151947/90334385-37722400-dfc5-11ea-8fd9-22db005e1dee.png)

Is there a way to do this in IM with a single command?

I'm using Imagemagick 6.9.10.23 on Ubuntu 20.04

**Comment by GeeMack** [comment, marked as answer]
With ImageMagick you can use  the "-coalesce" operation to turn all the frames of the GIF into single images, then the "-flatten" operation will flatten them all onto a single canvas. In the simplest form, a command like this should get you started...

     convert input.gif -background none -coalesce -flatten result.png

**Reply by hellocatfood** [reply]
Thank you! -flatten is what I needed
